### ISSUE TITLE
Icon picker improvements

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#488]
- [X] I have searched for existing issues to avoid creating duplicates

### TLDR
**The icon picker should have a search feature and allow choosing from all OUIF icons**

### Description
The icon picker today allows you to choose from 100 random OUIF icons.
This is a good intermediate solution and better than having no icons.

However, being able to specify an actual fitting icon to a customer/project is needed.

I suggest implementing an icon picker similar in UX to the macos emoji picker, but with OUIF icons

### Example

**pre search**
<img src="https://user-images.githubusercontent.com/7300548/92219241-97f4d280-ee9a-11ea-9bd0-61c3dcf5dbfc.png" width="350"  />

**on search**
<img src="https://user-images.githubusercontent.com/7300548/92219334-b529a100-ee9a-11ea-886c-8766d2aa5cb8.png" width="350" />

That is, a grid with e.g 8x8 icons with search-as-you-type and keyboard navigation.

See also https://uifabricicons.azurewebsites.net/
